tools/animfixer.py

```diff
@@ -4,7 +4,7 @@
 import argparse
 
 animdict ={
-    "SkelAnime_FrameUpdateMatrix":"SkelAnime_Update",
+    "SkelAnime_UpdateSkelAnime":"SkelAnime_Update",
```

Comment:
probably not an intended change
```suggestion
    "SkelAnime_FrameUpdateMatrix":"SkelAnime_Update",
```